pages/documentation/simulation/sensitivity/openlf.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+latex: true
+---
+
+# OpenLoadFlow
+
+PowSyBl OpenLoadFlow is an open-source power flow implementation in Java provided by PowSyBl. The source code is hosted on [GitHub](https://github.com/powsybl/powsybl-open-loadflow). 
```

Comment:
Maybe we should change **power flow implementation** by something about sensitivity analyses?

Reply:
I suggest to add this precision at the end : It also allows to perform sensitivity analyses.